ArticleTemplates/assets/js/bootstraps/article.js

```diff
@@ -1,17 +1,11 @@
 import { init as youtubeInit } from 'modules/youtube';
 import { init as twitterInit } from 'modules/twitter';
-import { init as initOutbrain } from 'modules/outbrain';
 import { init as quizInit } from 'modules/quiz';
 import { init as immersiveInit } from 'modules/immersive';
 import { init as creativeInjectorInit } from 'modules/creativeInjector';
 import { init as messengerInit } from 'modules/messenger';
 import resizeInit from 'modules/messenger/resize';
 
-function setupGlobals() {
-    window.articleOutbrainInserter = initOutbrain;
-    window.applyNativeFunctionCall('articleOutbrainInserter');
```

Comment:
There's one more use of this function in `mobile-apps-article-templates/ArticleTemplates/assets/js/boot.js`